CMakeLists is not included in pip bundle in pypi

See @utensil's [comment](https://github.com/numenta/nupic/issues/1097#issuecomment-49539900):

Gave it a try and encountered an error of missing `CMakeLists.txt`:

```
utensilsong@vm-1:~$ sudo pip install nupic 
Requirement already satisfied (use --upgrade to upgrade): nupic in /usr/local/lib/python2.7/dist-packages/nupic-_0.1.0_-py2.7.egg
Cleaning up...
utensilsong@vm-1:~$ sudo pip uninstall nupic                                                                                                                                                                                                                  
Uninstalling nupic:
  /usr/local/lib/python2.7/dist-packages/nupic-_0.1.0_-py2.7.egg
Proceed (y/n)? y
  Successfully uninstalled nupic
utensilsong@vm-1:~$ sudo pip install nupic                                                                                                                                                                                                                    
Downloading/unpacking nupic
  Downloading nupic-0.1.0.tar.gz (761kB): 761kB downloaded
  Running setup.py egg_info for package nupic

Installing collected packages: nupic
  Running setup.py install for nupic
    CMake Error: The source directory "/tmp/pip-build-root/nupic" does not appear to contain CMakeLists.txt.
    Specify --help for usage, or press the help button on the CMake GUI.
    Unable to generate build scripts!
    Complete output from command /usr/bin/python -c "import setuptools;__file__='/tmp/pip-build-root/nupic/setup.py';exec(compile(open(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-jJk5Jq-record/install-record.txt 
--single-version-externally-managed:
    CMake Error: The source directory "/tmp/pip-build-root/nupic" does not appear to contain CMakeLists.txt.

Specify --help for usage, or press the help button on the CMake GUI.

Unable to generate build scripts!

----------------------------------------
Command /usr/bin/python -c "import setuptools;__file__='/tmp/pip-build-root/nupic/setup.py';exec(compile(open(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-jJk5Jq-record/install-record.txt --single-version-external
ly-managed failed with error code 1 in /tmp/pip-build-root/nupic
Storing complete log in /home/utensilsong/.pip/pip.log
```

--- a/setup.py
+++ b/setup.py
@@ -94,7 +94,7 @@ def setup_nupic():
     version = properties['__version__'],
     packages = findPackages(repositoryDir),
     package_data = {
-      'nupic': ['README.md', 'LICENSE.txt', 'CMakeLists.txt', '.so', '.dll', '.dylib'],
+      'nupic': ['README.md', 'LICENSE.txt', 'CMakeLists.txt', '*.so', '*.dll', '*.dylib'],
       'nupic.bindings': ['_*.so', '_*.dll'],
       'nupic.data': ['*.json'],
       'nupic.frameworks.opf.exp_generator': ['*.json', '*.tpl'],

